src/js/0-global.js

```diff
@@ -153,7 +153,15 @@ function loadJSON(repo, filename, callback) {
       xobj.status != '200' && // if the status is NOT 'ok', remove the loading dots, and display an error:
       xobj.status != '0') { // for IE a cross domain request has status 0, we're going to execute this block fist, than the above as well.
         if (filename !== 'jck') {
-          document.getElementById('error-container').innerHTML = '<p>Error... there\'s a problem fetching the releases. Please see the <a href=\'https://github.com/AdoptOpenJDK/openjdk-releases/releases\' target=\'blank\'>releases list on GitHub</a>.</p>';
+          console.log(xobj.status)
```

Comment:
Do we need to log this?  If so then a more descriptive log line would be better.